### PR TITLE
bugfix/1617 - Add LXJ to fix KIAD spawn patterns

### LINKED ISSUE #1617: FlexJet (LXJ) used by KIAD, but doesn't exist
An airline is being referenced by KIAD spawn patterns, but the airline has not been created yet (LXJ - FlexJet).

### PR DESCRIPTION
**Pull request by j-651**
Resolves #1617 

The purpose of this pull request is to fix KIAD by adding Flexjet (LXJ), fixing Dulles' spawn patterns, and adding the necessary aircraft types for LXJ.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
 
 ### Bugfixes
+- <a href="https://github.com/openscope/openscope/issues/1617" target="_blank">#1617</a> - Add LXJ to fix crash at KIAD
 
 ### Enhancements & Refactors
 
 # 6.18.0 (July 1, 2020)
 ### New Features
-- <a href="https://github.com/openscope/openscope/issues/1236" target="_blank">#1523</a> - Re-introduce Washington Dulles International Airport (KIAD); add Mountain Air Cargo (MTN) and CommutAir (UCA)
+- <a href="https://github.com/openscope/openscope/issues/1523" target="_blank">#1523</a> - Re-introduce Washington Dulles International Airport (KIAD); add Mountain Air Cargo (MTN) and CommutAir (UCA)
```
Comment (erikquinn) [inline CHANGELOG.md:11]: You're killin me, @j-651...

Comment (j-651) [reply CHANGELOG.md:11]: I linked it to the issue as that was how the one below was done 😂

Comment (erikquinn) [reply CHANGELOG.md:11]: Oh god then the link was right and the text was wrong, and I changed the wrong one. Went from one right one wrong to both wrong.

Note that the issue number is BOTH in the link AND in the text that is shown to the user in the same line-- so you have to replace both.

I can fix it at the next pull request.

Comment (j-651) [reply CHANGELOG.md:11]: Yeah I don’t know why I only changed the link and left the text as it was when I copied it. That’s what happens when I try to do the changelog ahead of time!

Both should be #1236.

**Pull request merged**